prow/githuboauth/githuboauth.go

```diff
@@ -160,14 +161,17 @@ func (ga *GithubOAuthAgent) HandleRedirect(client OAuthClient, getter GithubClie
 			ga.serverError(w, "Save session", err)
 			return
 		}
-
 		ghc := getter.GetGithubClient(token.AccessToken, false)
 		user, err := ghc.GetUser("")
 		if err != nil {
 			ga.serverError(w, "Get user login", err)
 			return
 		}
-		http.SetCookie(w, &http.Cookie{Name: "github_login", Value: *user.Login})
+		http.SetCookie(w, &http.Cookie{
```

Comment:
please go ahead and add `Secure:   true`

Reply:
done